src/components/TextInput/index.tsx

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import { InputTitleText, InputText, InputDiv, ErrorText } from './styles';
 
+type TextInputProps = {
+  label: string;
+  placeholder: string;
+  erroring: boolean;
+  errorText: string;
+  type: string;
+  name: string;
+  value: string;
+  setValue: (newValue: string) => void;
```

Comment:
think typing this as a regular function should be fine for the most part but i'd recommend typing it as Dispatch<SetStateAction<string>> as that's the type of the setState function that is returned from useState

Reply:
Consulting @pragyakallanagoudar about this!

Reply:
implemented!